test_integration.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from bson.objectid import ObjectId
+from dotenv import find_dotenv, load_dotenv
+
+import app
+
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+
+    load_dotenv(file_path, override=True)
+
+    test_app = app.create_app()
+
+    test_app.testing = True
+
+    with test_app.test_client() as client:
+        yield client
+
+
+def test_example(client, monkeypatch):
+    monkeypatch.setattr('pymongo.MongoClient', mock_mongo)
```

Comment:
Nothing wrong with mocking it yourself, but if you want to add more integration tests you can use something like [mongomock](https://github.com/mongomock/mongomock) to avoid having to set it all up yourself.

Reply:
Yes, thanks I did look at this but my use case was so simple it was overkill. I will keep it in mind however